src/StyleableBannerWithLink/canRender.ts

```diff
@@ -7,13 +7,10 @@ import type { BrazeMessageProps } from './index';
 export const COMPONENT_NAME = 'StyleableBannerWithLink';
 
 export const canRender = (brazeMessageProps: BrazeMessageProps): boolean => {
-    const { header, body, buttonText, buttonUrl, imageUrl, imageAltText, ophanComponentId } =
-        brazeMessageProps;
+    const { header, body, buttonText, buttonUrl, imageUrl, ophanComponentId } = brazeMessageProps;
     if (imageUrl && !isImageUrlAllowed(imageUrl)) {
         console.log(`Image URL ${imageUrl} is not allowed`);
         return false;
     }
-    return Boolean(
-        header && body && buttonText && buttonUrl && imageUrl && imageAltText && ophanComponentId,
-    );
+    return Boolean(header && body && buttonText && buttonUrl && imageUrl && ophanComponentId);
```

Comment:
When imageAltText isn't provided, do we just not include any? Or is there a default? (I'm guessing we simply omit it).